packages/common/src/dto/skill-information.dto.ts

```diff
@@ -73,7 +73,7 @@ export class SkillInformationDTO {
   @ValidateIf(o => EmploymentTypes.NOT_HEALTH_SECTOR_EMPLOYED === o.currentEmployment)
   @IsIn(Object.values(EmploymentCircumstances), { message: 'Invalid circumstance selection' })
   @IsString({ message: 'Circumstance selection is required' })
-  employmentCircumstance!: EmploymentCircumstances;
+  employmentCircumstance!: EmploymentCircumstances | undefined;
```

Comment:
```suggestion
  employmentCircumstance?: EmploymentCircumstances;
```

Does this make more sense? or is there a difference

Reply:
At this point I think it's equivalent.

Reply:
Seems weird to declare it non-null and then say it can be undefined :P